wandb/sdk/launch/docker.py

```diff
@@ -56,44 +57,43 @@ def generate_docker_image(project: _project_spec.Project, entry_cmd):
     stderr = subprocess.run(
         cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE
     ).stderr.decode("utf-8")
+    print(stderr)
```

Comment:
This'll only print stuff after the process is done running. Would be cool to tee live output.

Reply:
But not necessary for this PR.